priv/repo/migrations/00000000000001_create_user.exs

```diff
@@ -9,7 +9,6 @@ defmodule Org.Repo.Migrations.CreateUser do
       add :login, :string
       add :company, :string
       add :location, :string
-      add :created_at, :string
```

Comment:
I think I named it that way because that's how it's named in the github response, so for ease of use. I don't mind changing it, but we have to consider how it will affect all our existing user data.